### DISCUSSION TITLE
dotnet-sqltest create test coverage report

### DISCUSSION THREAD

**Discussion by rapolas-oag** [cagrin/dotnet-sqltest#18] [Ideas]
As *dotnet-sqltest* creates test execution report and it shows not covered by test code lines in CLI, I think it would be nice to have code coverage report as output of the cli run which could be uploaded to build running agent (Azure DevOps build).

**Comment by cagrin** [comment]
Code coverage is based on https://github.com/GoEddie/SQLCover. I did some custom work to make it Azure SQL Edge compatible. It is possible to generate coverage report from SQLCover itself.

I can wrap on top of SQLCoverLib and make it very convenient to use in dotnet-sqltest tool. What format of coverage report do you prefer?

**Reply by rapolas-oag** [reply]
I would prefer Cobertura support to be out of the box.

**Comment by cagrin** [comment]
I have just published dotnet-sqltest 1.4.0 with ```--cc-cobertura <file>``` option. Hope you enjoy :-)

**Reply by rapolas-oag** [reply]
Testing...

**Reply by rapolas-oag** [reply]
I love the tool, thank you!
